cfg/config.yaml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
All the changes as part of this file have already been applied to master by other PRs, so it can be safely removed as part of this PR.